base64.cabal

```diff
@@ -57,7 +57,7 @@ library
       base           >=4.14     && <4.17
     , bytestring     >=0.10     && <0.12
     , deepseq        >=1.4.3.0  && <1.5
-    , text           ^>=1.2
+    , text           >=1.2      && <3
```

Comment:
Tight bound please :)

Reply:
You mean `>=1.2 && <1.3 || >=2.0 && <2.1`, similar to text-short?

Reply:
yeh, perfect